scheduler/plugin/host.go

```diff
@@ -172,6 +172,15 @@ type stack struct {
 
 	// resultNormalizedScoreList is returned by guest.normalizedscoreFn
 	resultNormalizedScoreList framework.NodeScoreList
+
+	// podInfoToAdd is used by guest.addpodFn
+	podInfoToAdd *v1.Pod
+
+	// podInfoToAdd is used by guest.addpodFn
+	podInfoToRemove *v1.Pod
+
+	// podInfoToRemove is used by guest addpodFn & guest.removepodFn
+	nodeInfo *v1.Node
```

Comment:
```suggestion
```

You don't need to define nodeInfo here. You can reuse node again same as Filter.